main.tf

```diff
@@ -59,6 +59,16 @@ resource "azurerm_log_analytics_solution" "logs" {
   }
 }
 
+data "azurerm_subscription" "current" {
+  subscription_id = var.subscription_id
+}
+
+resource "azurerm_role_assignment" "sp" {
+  scope                = data.azurerm_subscription.current.id
+  role_definition_name = "Network Contributor"
```

Comment:
Do we need permissions to read / write from the storage containers?

Reply:
The Domino stands up fine with this permission. The storage containers access may already be a part of the Managed Identity?

Reply:
If you can login successfully, view the readme on the quick-start project and can build an environment that should sufficiently prove that containers work.